src/data/list/basic.lean

```diff
@@ -3222,6 +3243,48 @@ theorem mem_bag_inter {a : α} : ∀ {l₁ l₂ : list α}, a ∈ l₁.bag_inter
       rintro ⟨rfl, h'⟩, exact h.elim h' }
   end
 
+@[simp] theorem count_bag_inter {a : α} :
```

Comment:
Was this proof auto-generated? It could use a lot of cleaning up. In particular, the many non-terminal `simp` calls don't look good to me. Can the `rw _, rw_` lines be combined into one, or are they separate goals? (If so, can this be made clearer?)

Reply:
Combined `rw`s and replaced `simp`s with `simp only`s.